README.md

```diff
@@ -506,6 +506,26 @@ extra conditions contradict the conditions in the rules, the Transfer can never
 The Receiver must post a comment describing the resolution of these extra conditions with validity as a result, before
 he can accept the Transfer.
 
+# Representation of Nomics
+
+## Mutable Rules
+
+**377** *Representation Theory*
+
+A **Nomic** is a game in which the rules may change. 
```

Comment:
Is that so? I mean, sure those are currently the rules, but they don't always have to be? There is nothing that prevents us from throwing out all the rules that say rule changes are allowed after turning it in to monopoly, right?